docs/gpa.md

```diff
@@ -233,78 +231,23 @@ The information included in the notice of intended procurement covers all the in
     <tbody>
       <tr class="section">
         <td>X:7</td>
-        <td colspan="2">A procuring entity shall make available to suppliers tender documentation that includes all information necessary to permit suppliers to prepare and submit responsive tenders. Unless already provided in the notice of intended procurement, such documentation shall include a complete description of:</td>
-      </tr>
-      <tr>
-        <td>X:7(a)</td>
-        <td>the procurement, including the nature and the quantity of the goods or services to be procured or, where the quantity is not known, the estimated quantity and any requirements to be fulfilled, including any technical specifications, conformity assessment certification, plans, drawings or instructional materials;</td>
```

Comment:
Let's either repeat or refer back to the earlier mapping. You can add `id="VII:2"` etc. attributes so that we can add links. In the EU profile, we wrote "Same as … above" and then repeated the mapping, like at the bottom of [F01](https://standard.open-contracting.org/profiles/eu/master/en/forms/F01/).

Reply:
I will update all the cross-references at once.

Reply:
Okay, I'll create a new issue so that we can close this one sooner :) #23